x-pack/plugins/security_solution/common/endpoint/data_loaders/index_fleet_endpoint_policy.ts

```diff
@@ -106,18 +114,48 @@ export const indexFleetEndpointPolicy = usageTracker.track(
         version: packageVersion,
       },
     };
-    const packagePolicy = (await kbnClient
-      .request({
-        path: PACKAGE_POLICY_API_ROUTES.CREATE_PATTERN,
-        method: 'POST',
-        body: newPackagePolicyData,
-        headers: {
-          'elastic-api-version': API_VERSIONS.public.v1,
-        },
-      })
-      .catch(wrapErrorAndRejectPromise)) as AxiosResponse<CreatePackagePolicyResponse>;
 
-    response.integrationPolicies.push(packagePolicy.data.item as PolicyData);
+    const createPackagePolicy = async (): Promise<CreatePackagePolicyResponse> =>
+      kbnClient
+        .request<CreatePackagePolicyResponse>({
+          path: PACKAGE_POLICY_API_ROUTES.CREATE_PATTERN,
+          method: 'POST',
+          body: newPackagePolicyData,
+          headers: {
+            'elastic-api-version': API_VERSIONS.public.v1,
+          },
+        })
+        .catch(catchAxiosErrorFormatAndThrow)
+        .then((res) => res.data);
+
+    const started = new Date();
+    const hasTimedOut = (): boolean => {
+      const elapsedTime = Date.now() - started.getTime();
+      return elapsedTime > TimeoutsInMS.FIVE_MINUTES;
+    };
+
+    let packagePolicy: CreatePackagePolicyResponse | undefined;
+    log?.info(`Creating package policy for ${policyName}`);
```

Comment:
Does this need to be a `.info()` or should it be `.debug()`?

also - suggestion change to the message: `Creating integration policy with name: ${policyName}`

Reply:
Changed